src/app/OrderBook.tsx

```diff
@@ -0,0 +1,20 @@
+import { useContext } from "react";
+import { AdexStateContext } from "./page";
+
+export function OrderBook({ orders }) {
```

Comment:
here there is still an error: "Binding element 'orders' implicitly has an 'any' type.ts(7031)"

Reply:
Specified the type, shouldn't occur now